tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -74,6 +123,14 @@ describe(__filename, () => {
     );
   };
 
+  const renderWithCurrentVersion = ({ ...props } = {}) => {
+    _loadVersions({
```

Comment:
Hmm, I would have expected this to accept an `addon` argument to be used with `_loadVersions`, but I guess every call to load versions works with the default `fakeAddon`? It still seems safer to me to have this accept an `addon` and use that for `_loadVersions`.

Reply:
Note that my final suggestion about having a global `addon` object (which this function could use), would address the above concern without needing the argument.